### ISSUE TITLE
Alignment of headings (ActionText) in actions menu

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud-vue#1044]
Currently the "Chat notifications" heading in the actions menu of the navigation is not really aligned anywhere:
![chat notifications menu](https://user-images.githubusercontent.com/925062/71559803-0922f100-2a95-11ea-9119-6f4fe985c18a.png)

It should move slightly to the right to be aligned with the other text. :)

**Comment by jancborchardt** [comment]
→ Moved to Vue components and reopening.

**Comment by raimund-schluessler** [comment]
This still seems to be an issue, but different than before. See here:

![Screenshot 2023-01-27 at 12-35-46 Nextcloud Vue Style Guide](https://user-images.githubusercontent.com/2496460/215077176-d155d777-d57b-46ea-a394-9578d39ff50e.png)

@jancborchardt @nimishavijay  Could you check how this should look like, pleaes?

**Comment by nimishavijay** [comment]
You're right @raimund-schluessler, now it's too far to the left. What @jancborchardt meant was left aligning all the text. `ActionText without icon` should be moved right so that it is aligned with `ActionText with icon` and the other items. Perhaps `ActionText without icon` should have a higher left margin?
